omnibus/config/projects/agent.rb

```diff
@@ -380,6 +380,24 @@ def generate_embedded_archive(version)
 exclude 'bundler\/git'
 
 if windows?
+  FORBIDDEN_SYMBOLS = [
+    "glog.init"
+  ]
+  
+  raise_if_forbidden_symbol_found = Proc.new { |symbols|
+    FORBIDDEN_SYMBOLS.each do |fs|
+      count = symbols.scan(fs).count()
```

Comment:
Couldn't this provide false positive, for instance wouldn't `somethinglog.init` would be matched no?

Reply:
Sounds like a good idea IMO!